src/examples/index.ts

```diff
@@ -38,7 +38,7 @@ export async function loadExamples(): Promise<Record<string, Example>> {
       },
     ),
     ...Object.entries(
-      import.meta.glob("./**/*.{txt,js,cds}", {
+      import.meta.glob("./**/*.{txt,js,cds,csv}", {
```

Comment:
Do you also want to add csv files? It doesn't seem to be included in this PR.

Reply:
We do add _.csv_ files, but implicitly. ☺️ Our documentation rule examples (see the screenshot for the rule that lints csv) each *compute the link* to the playground, thereby providing the source files without us having to check them in here.